src/main/java/org/gridsuite/filter/server/LoadFilterRepositoryProxy.java

```diff
@@ -38,17 +44,32 @@ public LoadFilterRepository getRepository() {
 
     @Override
     public AbstractFilter toDto(LoadFilterEntity entity) {
-        return buildInjectionFilter(
-            LoadFilter.builder(), entity).build();
+        return new FormFilter(
```

Comment:
shouldn't we have something like
toDto(LoadFilterEntity entity) { 
   return new FormFilter( entity.getId(), entity.getCreationDate(), entity.getModificationDate(), createSpecificFilter());
}
in the parent interface/class
and createSpecificFilter in specific implementation ?

Reply:
Done. And I added an InjectionAttributes class to build AbstractInjectionFilter with more genericity